src/controller/stream-controller.ts

```diff
@@ -49,7 +49,7 @@ import type {
 import type { Level } from '../types/level';
 import type { Track, TrackSet } from '../types/track';
 import type { TransmuxerResult } from '../types/transmuxer';
-import type { BufferInfo } from '../utils/buffer-helper';
+import type { BufferInfo, BufferTimeRange } from '../utils/buffer-helper';
```

Comment:
The build is failing because this import was not removed

```suggestion
import type { BufferInfo } from '../utils/buffer-helper';
```